tgfilestream/web_routes.py

```diff
@@ -63,8 +63,21 @@ async def handle_request(req: web.Request, head: bool = False) -> web.Response:
         return web.Response(status=404, text="404: Not Found")
 
     size = message.file.size
-    offset = req.http_range.start or 0
-    limit = req.http_range.stop or size
+    try:
+        offset = req.http_range.start or 0
+        limit = req.http_range.stop or size
+        if (limit > size) or (offset < 0) or (limit < offset):
+            raise ValueError("range not in acceptable format")
+    except ValueError:
+        return web.Response(
+            status=416,
+            text="416: Range Not Satisfiable",
+            headers = {
+                "Content-Range": f"bytes */{size}"
+            }
+        )
+    
+    log.info(f"Range {offset} - {limit}")
```

Comment:
This shouldn't be a separate log line, could maybe be integrated into the existing "Serving file" log